SCons/Util.py

```diff
@@ -1233,11 +1232,11 @@ def default_idfun(x):
 def uniquer_hashables(seq):
     seen = {}
     result = []
+    result_append = result.append
```

Comment:
Add a comment here as to the why of this?

Reply:
Sure. I forgot this was in the changset, I seem to have parts of two. Can also back this out and make separate.